### PR TITLE
feat: update explict support for new clientjs default timeout

### PR DESCRIPTION
**Pull request by zcstarr**

### CODE REVIEW

**Comment by shanejonas** [comment]
we need to add `onError` and `onNotification` pass-throughs i think

**Comment by zcstarr** [comment]
> we need to add `onError` and `onNotification` pass-throughs i think

100% forgot about those :D

File: templates/typescript/templated/exported-class.template.ts [hunk 1]
```diff
@@ -96,13 +128,17 @@ export class <%= className %> {
     } else {
       rpcParams = params;
     }
-    return this.rpc.request(methodName, rpcParams);
+    if(notification) {
```
Comment (shanejonas) [inline templates/typescript/templated/exported-class.template.ts]:
```suggestion
    if (notification) {
```

**Pull request merged**